README.md

```diff
@@ -1,4 +1,7 @@
 # Reticulum
+Note: **Due to our small team size, we don't support setting up reticulum locally due to restrictions on developer credentials. You're welcome to set up this up youself. In addition to runing reticulum, you'll need to also run hubs and dialog locally (because our developer dialog server is locked down)**
```

Comment:
runing -> running

Reply:
```suggestion
Note: **Due to our small team size, we don't support setting up Reticulum locally due to restrictions on developer credentials. Although relatively difficult and new territory, you're welcome to set up this up yourself. In addition to running Reticulum, you'll need to also run [Hubs](https://github.com/mozilla/hubs) and [Dialog](https://github.com/mozilla/dialog) locally because the developer Dialog server is locked down and your local Reticulum will not connect properly.**
```